integreat_compass/cms/templates/offers/offer_form.html

```diff
@@ -0,0 +1,132 @@
+                                                        <label for="{{ choice.id_for_label }}"
+                                                               class="w-full duration-100 m-0 p-2 text-lg rounded-full peer-checked:bg-primary peer-checked:text-white">
+                                                            {{ choice.choice_label }}
+                                                        </label>
+                                                    </div>
+                                                </li>
+                                            {% endfor %}
+                                        </ul>
+                                    </div>
+                                {% endif %}
+                            {% endfor %}
+                        </div>
+                    </div>
+                    <div>
+                        {% for field in location_form %}
+                            <div>
+                                <label for="{{ field.id_for_label }}"
+                                       {% if field.field.required %}class="field-required"{% endif %}>
+                                    {{ field.label }}
+                                </label>
+                                {% render_field field %}
+                                <div class="help-text">{{ field.help_text }}</div>
+                            </div>
+                        {% endfor %}
+                    </div>
+                </div>
+                <div class="flex flex-col gap-8">
+                    <div>
+                        <h2>{% translate "Contact information" %}</h2>
+                        {% for field in contact_form %}
+                            <div>
+                                <label for="{{ field.id_for_label }}"
+                                       {% if field.field.required %}class="field-required"{% endif %}>
+                                    {{ field.label }}
+                                </label>
+                                {% render_field field %}
+                                <div class="help-text">{{ field.help_text }}</div>
+                            </div>
+                        {% endfor %}
+                    </div>
+                    <div>
+                        <h2>{% translate "Information of organization" %}</h2>
+                        {% for field in organization_form %}
+                            <div>
+                                <label for="{{ field.id_for_label }}"
+                                       {% if field.field.required %}class="field-required"{% endif %}>
+                                    {{ field.label }}
+                                </label>
+                                {% render_field field %}
+                                <div class="help-text">{{ field.help_text }}</div>
+                            </div>
+                        {% endfor %}
```

Comment:
```suggestion
                        {{ organization_form }}
```
If you're not rendering the fields individually, you can also go with the default Django rendering... this works for the other forms as well.
(just rename the `help-text` class to `helptext` which is the default class assigned to it)

Reply:
Will this use the `WIDGET_ERROR_CLASS` though? I think that is widget-tweaks specific, right? Also the additional class on the label fir required fields.

I know it is a bit more verbose, but it's probably easier to keep everything consistent?

Reply:
Ah, right :+1: